### PR TITLE
Syslog sender 5.5

### PR DESCRIPTION
**Pull request by liorblob**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20605
fixes: https://github.com/demisto/etc/issues/11365

## Description
Allow sending messages and mirroring investigations to syslog

## Required version of Demisto
5.5

## Must have
- [X] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.

- [ ] [README](https://github.com/demisto/content/compare/syslog-sender-5.5?expand=1#diff-2515e91d1cebf50ae3739a9be3d92916)
- [ ] [YAML file](https://github.com/demisto/content/pull/5318/files#diff-e7398a92df97a93a1ad4e158caae4fa2)

### CODE REVIEW

**Comment by liorblob** [comment]
@kirbles19 we changes some things, you covered the most, but I guess this would take going over one more time. Also there's the README file.

File: Packs/Syslog/Integrations/SyslogSender/SyslogSender.yml [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+      - LOG_LOCAL5
+      - LOG_LOCAL6
+      - LOG_LOCAL7
+      required: false
+      secret: false
+    deprecated: false
+    description: Send a message to Syslog.
+    execution: false
+    name: syslog-send
+  dockerimage: demisto/python3:3.7.4.977
```
Comment (glicht) [inline Packs/Syslog/Integrations/SyslogSender/SyslogSender.yml]: Use the latest docker image

**Comment by kirbles19** [comment]
@liorblob Done.

**Comment by dorsha** [comment]
@liorblob / @glicht what is the status of this PR?

**Comment by liorblob** [comment]
@Itay4 I think we need to review again after the version change

File: Packs/Syslog/Integrations/SyslogSender/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## Use Cases
+---
+* Send messages to Syslog via TCP or UDP.
+* Mirror incident war room entries to Syslog.
+
+## Configure Syslog Sender on Demisto
+---
+To allow sending messages to syslog via Demisto, the following lines have to be in the Syslog configuration:
+
```
Comment (glicht) [inline Packs/Syslog/Integrations/SyslogSender/README.md]: This statement is not accurate. The configuration below is for rsyslog.

File: Packs/Syslog/Integrations/SyslogSender/SyslogSender.py [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+
+def init_manager(params: dict) -> SyslogManager:
+    """
+    Create a syslog manager instance according to provided parameters.
+    :param params: Parameters for the syslog manager.
+    :return: syslog manager
+    """
+    address = params.get('address', '')
+    port = int(params.get('port', 514))
+    protocol = params.get('protocol', 'udp').lower()
```
Comment (glicht) [inline Packs/Syslog/Integrations/SyslogSender/SyslogSender.py]: In line 136 you are using uppercase UDP. Need to be consistent. Probably best to use constants.

**Comment by liorblob** [comment]
@glicht not sure about this test because we need to specify the IP address of the docker container in the instance configuration of syslog in test-conf. Also not sure how to verify that in a test playbook...
Other than that I fixed the other things.

**Comment by glicht** [comment]
Let's add DemistoServerURL to Content not just test. I can see a general use for it. Add also to the output FullURL.

File: Packs/GetServerURL/README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+## GetServerURL
+Get the Cortex XSOAR Server URL.
```
Comment (Itay4) [inline Packs/GetServerURL/README.md]: lets get rid of the xsoar here
script doc should look something like the ones here https://github.com/ElazarK/content-docs/tree/master/scripts

File: Packs/Syslog/Integrations/SyslogSender/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+## Unreleased
```
Comment (Itay4) [inline Packs/Syslog/Integrations/SyslogSender/CHANGELOG.md]: remove changelog, the build should generate it

**Pull request merged**